gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa/factors-select-dialog/factors-select-dialog.component.ts

```diff
@@ -70,17 +71,10 @@ export class FactorsSelectDialogComponent implements OnInit {
     return this.factors.map((factor) => this.model.get(factor.id)).filter((selected) => selected).length === 0;
   }
 
-  cancel(): void {
+  closeDialog(): void {
     this.dialogRef.close();
```

Comment:
would it be better to close first and then navigate to a new link? I am not sure if the closeDialog() is executed at all 

this.router.navigate([this.mfaSettingsLink]);